src/server/api/user/getInvitations.ts

```diff
@@ -0,0 +1,19 @@
+import { Response } from 'express'
+
+import { CycleRequest } from 'meta/api/request'
+
+import { AssessmentController } from 'server/controller/assessment'
+import { UserController } from 'server/controller/user'
+import { Requests } from 'server/utils'
+
+export const getInvitations = async (req: CycleRequest, res: Response) => {
```

Comment:
Get invitations for country home - works same as admin but we pass `countryIso` and `pendingOnly: true`